<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comparando</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tamanho do vetor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Método de preenchimento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Algoritmo de ordenação:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tamanho">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="tipoVetor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aleatório</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Crescente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrescente</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bubble</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shell</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="criaPreenche">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar vetor e preencher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ordenar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar vetor</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="info">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="choose">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Para iniciar, escolha uma opção no MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grafico">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar Gráfico</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <zorder>choose</zorder>
   <zorder>titulo</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>tamanho</zorder>
   <zorder>tipoVetor</zorder>
   <zorder>sort</zorder>
   <zorder>criaPreenche</zorder>
   <zorder>ordenar</zorder>
   <zorder>info</zorder>
   <zorder>grafico</zorder>
   <zorder>plot</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionMexer_com_ordena_es"/>
    <addaction name="actionGr_fico"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMexer_com_ordena_es">
   <property name="text">
    <string>Vetor</string>
   </property>
  </action>
  <action name="actionGr_fico">
   <property name="text">
    <string>Gráfico</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
